### ISSUE TITLE
puppetserver 7 warning: file does not contain a valid yaml hash

### ISSUE THREAD

**Issue by jhoblitt** [camptocamp/puppet-augeas#91]
When using this module under puppetserver7, this warning is logged:
```
2022-10-13T16:20:42.371Z WARN  [qtp756172032-476376] [puppetserver] Puppet /etc/puppetlabs/code/environments/IT_3057_el8/modules/augeas/data/common.yaml: file does not contain a valid yaml hash
```

**Comment by jhoblitt** [comment]
Ping?

**Comment by Pigueiras** [comment]
It's fixed in #85, so if you use the module from master you shouldn't see the warning.
